README.md

```diff
@@ -211,7 +211,7 @@ When doing multiple runs ensure you retain any authentication artefacts you need
 Existing tests can be run using
 
 ```bash
-ansible-playbook tests/run_tests.yml
+ANSIBLE_INVENTORY_UNPARSED_FAILED=false ansible-playbook tests/run_tests.yml
```

Comment:
Would adding the `unparsed_is_failed` setting to `tests/ansible.cfg` break the execution of our tests?
If we were to add this setting to the config file, we could then document that the official way of running tests is to use `ansible-playbook run_tests.yml` executed from the `tests/` directory, instead of requiring an environment variable to cover for an implementation detail.

Reply:
Good point I'll have a look but I think it should be fine.